api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```diff
@@ -189,12 +189,26 @@ public boolean isUsernameTaken(String username) {
   }
 
   // Returns a user's contact email address via the custom schema in the directory API.
-  public String getContactEmailAddress(String username) {
-    return (String)
-        getUserByUsername(username)
-            .getCustomSchemas()
-            .get(GSUITE_AOU_SCHEMA_NAME)
-            .get(GSUITE_FIELD_CONTACT_EMAIL);
+  public Optional<String> getContactEmail(String username) {
+    return Optional.ofNullable(getUserByUsername(username))
+        .map(
+            user ->
+                (String)
```

Comment:
nit: If we need to do much with this API, a helper class might...help. For example
```
Optional<String> getStringField(customSchemaMap, schemaName, fieldName)
```